ptero_workflow/api/v1/api.py

```diff
@@ -12,6 +12,14 @@
 api.add_resource(views.WorkflowDetailView,
     '/workflows/<int:workflow_id>', endpoint='workflow-detail')
 
+api.add_resource(views.ExecutionInputsView,
+    '/executions/<int:execution_id>/inputs',
+    endpoint='execution-inputs')
+
+api.add_resource(views.ExecutionOutputsView,
```

Comment:
The biggest difference that I see between having a single endpoint and two endpoints is that one requires mutable endpoints and the other supports immutable endpoints.